src/main.ts

```diff
@@ -39,6 +41,12 @@ Vue.use(Vue2Filters)
 //
 fetchConfig()
   .then(async () => {
+    // initialize Sentry
+    Sentry.init({
```

Comment:
Should this be "await" ?

Note that this code has evolved a bit since this, but it should be easy to merge it.

Also the convention is to display a message before doing something (eg, "Initializing Sentry...").

Also you need to decorate your console.info to preempt a lint error (see other console.log elsewhere for example).

Reply:
Ah okay. I don't it needs an await - I tested it by locally throwing errors and it was sending me sentry error messages